tests/conftest.py

```diff
@@ -224,35 +225,59 @@ def add_permissions(func, path, exc_info):
         raise
 
 
+@pytest.fixture
+def no_git() -> bool:
+    """Give the `no_git` parameter for any fixtures that uses it a default value
+
+    By default, git wil always be enabled.
+
+    Some fixtures (like `temp_testbed`) have the option to run without doing any git
+    related actions. In tests that use these fixtures, git can be disabled like so:
+
+        @pytest.mark.parametrize("no_git", [True])
+        def test_some_filesystem_stuff(temp_testbed):
+            ...
+
+    If this pytest mark is omitted, git will be enabled for any fixtures that have
+    the `no_git` parameter:
+
+        def test_some_filesystem_stuff(temp_testbed):
+            ...
+    """
+    return False
+
+
 @pytest.fixture(autouse=True)
-def temp_testbed(monkeypatch, get_marks):
+def temp_testbed(monkeypatch, get_marks, no_git):
     # create temporary copy of testbed, complete with git repo
     # realpath() resolves symlinks, required for paths to match on macOS
     temp_dir = os.path.realpath(tempfile.mkdtemp())
     temp_testbed = os.path.join(temp_dir, "testbed")
     os.mkdir(temp_testbed)
 
-    # Initialize git repo
-    run_git_command(temp_testbed, "init")
+    if no_git is not True:
```

Comment:
Would it be simpler to add `no_git` the way `clear_testbed` is added, and then just get the value from get_marks? 

Then instead of `pytest.parametrize(...)`, just `pytest.mark.no_git`.

Reply:
I agree, I think this would be a lot cleaner

Reply:
Nice yea will change this